[2025-05-12T17:17:19.477+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gold_pipeline.load_gold_data manual__2025-05-12T17:16:19.075075+00:00 [queued]>
[2025-05-12T17:17:19.490+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gold_pipeline.load_gold_data manual__2025-05-12T17:16:19.075075+00:00 [queued]>
[2025-05-12T17:17:19.491+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2025-05-12T17:17:19.510+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): load_gold_data> on 2025-05-12 17:16:19.075075+00:00
[2025-05-12T17:17:19.517+0000] {standard_task_runner.py:57} INFO - Started process 1939 to run task
[2025-05-12T17:17:19.520+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'gold_pipeline', 'load_gold_data', 'manual__2025-05-12T17:16:19.075075+00:00', '--job-id', '79', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp4ailyuep']
[2025-05-12T17:17:19.526+0000] {standard_task_runner.py:85} INFO - Job 79: Subtask load_gold_data
[2025-05-12T17:17:19.580+0000] {task_command.py:415} INFO - Running <TaskInstance: gold_pipeline.load_gold_data manual__2025-05-12T17:16:19.075075+00:00 [running]> on host 51d6065cf9f5
[2025-05-12T17:17:19.713+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gold_pipeline' AIRFLOW_CTX_TASK_ID='load_gold_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T17:16:19.075075+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T17:16:19.075075+00:00'
[2025-05-12T17:17:21.881+0000] {logging_mixin.py:151} INFO - Permissions définies avec succès sur /gold_datalake/processed (777)
[2025-05-12T17:17:21.882+0000] {client.py:724} INFO - Reading file '/gold_datalake/processed/gold_prices_2025-05-12.parquet'.
[2025-05-12T17:17:21.945+0000] {logging_mixin.py:151} INFO - Fichier Parquet lu avec succès depuis HDFS: /gold_datalake/processed/gold_prices_2025-05-12.parquet
[2025-05-12T17:17:21.945+0000] {client.py:724} INFO - Reading file '/gold_datalake/processed/dim_date_2025-05-12.parquet'.
[2025-05-12T17:17:21.970+0000] {logging_mixin.py:151} INFO - Fichier Parquet lu avec succès depuis HDFS: /gold_datalake/processed/dim_date_2025-05-12.parquet
[2025-05-12T17:17:21.970+0000] {client.py:724} INFO - Reading file '/gold_datalake/processed/dim_marche_2025-05-12.parquet'.
[2025-05-12T17:17:21.994+0000] {logging_mixin.py:151} INFO - Fichier Parquet lu avec succès depuis HDFS: /gold_datalake/processed/dim_marche_2025-05-12.parquet
[2025-05-12T17:17:22.017+0000] {connection.py:314} INFO - Snowflake Connector for Python Version: 3.1.1, Python Version: 3.8.18, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.2.5
[2025-05-12T17:17:22.018+0000] {connection.py:1050} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-05-12T17:17:22.019+0000] {connection.py:1068} INFO - Setting use_openssl_only mode to False
[2025-05-12T17:17:22.879+0000] {cursor.py:804} INFO - query: [select current_database(), current_schema();]
[2025-05-12T17:17:22.988+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:22.988+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:22.990+0000] {cursor.py:804} INFO - query: [ROLLBACK]
[2025-05-12T17:17:23.101+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:23.102+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:23.109+0000] {cursor.py:804} INFO - query: [DESC TABLE /* sqlalchemy:_has_object */ "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_DATE"...]
[2025-05-12T17:17:23.204+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:23.234+0000] {cursor.py:804} INFO - query: [CREATE TABLE "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_DATE" ( date_id datetime, jour I...]
[2025-05-12T17:17:23.404+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:23.405+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:23.430+0000] {cursor.py:804} INFO - query: [INSERT INTO "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_DATE" (date_id, jour, mois, annee...]
[2025-05-12T17:17:23.877+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:23.879+0000] {cursor.py:804} INFO - query: [SHOW /* sqlalchemy:get_table_names */ TABLES IN "GOLD_ANALYSIS"."MARCHE"]
[2025-05-12T17:17:23.994+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:23.996+0000] {cursor.py:959} INFO - Number of results in first chunk: 4
[2025-05-12T17:17:23.998+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/load.py:168 UserWarning: The provided table name 'TEMP_DIM_DATE' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.
[2025-05-12T17:17:23.999+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:24.188+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:24.189+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:24.189+0000] {cursor.py:804} INFO - query: [MERGE INTO GOLD_ANALYSIS.MARCHE.DIM_DATE AS target USING GOLD_ANALYSIS.MARCHE.TE...]
[2025-05-12T17:17:24.604+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:24.605+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:24.789+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:24.790+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:24.791+0000] {cursor.py:804} INFO - query: [DROP TABLE IF EXISTS GOLD_ANALYSIS.MARCHE.TEMP_DIM_DATE]
[2025-05-12T17:17:24.928+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:24.929+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:24.930+0000] {logging_mixin.py:151} INFO - Table Dim_Date mise à jour avec succès dans Snowflake (sans doublons)
[2025-05-12T17:17:24.934+0000] {cursor.py:804} INFO - query: [DESC TABLE /* sqlalchemy:_has_object */ "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_MARCH...]
[2025-05-12T17:17:25.029+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:25.032+0000] {cursor.py:804} INFO - query: [CREATE TABLE "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_MARCHE" ( date_id datetime, volu...]
[2025-05-12T17:17:25.221+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:25.222+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:25.250+0000] {cursor.py:804} INFO - query: [INSERT INTO "GOLD_ANALYSIS"."MARCHE"."TEMP_DIM_MARCHE" (date_id, volume_sp500, m...]
[2025-05-12T17:17:25.688+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:25.691+0000] {cursor.py:804} INFO - query: [SHOW /* sqlalchemy:get_table_names */ TABLES IN "GOLD_ANALYSIS"."MARCHE"]
[2025-05-12T17:17:25.807+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:25.808+0000] {cursor.py:959} INFO - Number of results in first chunk: 4
[2025-05-12T17:17:25.811+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/load.py:189 UserWarning: The provided table name 'TEMP_DIM_MARCHE' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.
[2025-05-12T17:17:25.812+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:26.024+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:26.025+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:26.026+0000] {cursor.py:804} INFO - query: [MERGE INTO GOLD_ANALYSIS.MARCHE.DIM_MARCHE AS target USING GOLD_ANALYSIS.MARCHE....]
[2025-05-12T17:17:26.477+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:26.478+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:26.688+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:26.689+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:26.690+0000] {cursor.py:804} INFO - query: [DROP TABLE IF EXISTS GOLD_ANALYSIS.MARCHE.TEMP_DIM_MARCHE]
[2025-05-12T17:17:26.813+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:26.814+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:26.814+0000] {logging_mixin.py:151} INFO - Table Dim_Marche mise à jour avec succès dans Snowflake (sans doublons)
[2025-05-12T17:17:26.817+0000] {cursor.py:804} INFO - query: [DESC TABLE /* sqlalchemy:_has_object */ "GOLD_ANALYSIS"."MARCHE"."TEMP_FAIT_PRIX...]
[2025-05-12T17:17:26.922+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:26.927+0000] {cursor.py:804} INFO - query: [CREATE TABLE "GOLD_ANALYSIS"."MARCHE"."TEMP_FAIT_PRIX_OR" ( date_id datetime, pr...]
[2025-05-12T17:17:27.078+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:27.079+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:27.111+0000] {cursor.py:804} INFO - query: [INSERT INTO "GOLD_ANALYSIS"."MARCHE"."TEMP_FAIT_PRIX_OR" (date_id, prix_or, volu...]
[2025-05-12T17:17:27.607+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:27.609+0000] {cursor.py:804} INFO - query: [SHOW /* sqlalchemy:get_table_names */ TABLES IN "GOLD_ANALYSIS"."MARCHE"]
[2025-05-12T17:17:27.716+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:27.718+0000] {cursor.py:959} INFO - Number of results in first chunk: 4
[2025-05-12T17:17:27.720+0000] {logging_mixin.py:151} WARNING - /opt/***/dags/load.py:207 UserWarning: The provided table name 'TEMP_FAIT_PRIX_OR' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.
[2025-05-12T17:17:27.722+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:27.958+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:27.959+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:27.961+0000] {cursor.py:804} INFO - query: [MERGE INTO GOLD_ANALYSIS.MARCHE.FAIT_PRIX_OR AS target USING GOLD_ANALYSIS.MARCH...]
[2025-05-12T17:17:28.391+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:28.392+0000] {cursor.py:804} INFO - query: [COMMIT]
[2025-05-12T17:17:28.614+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:28.615+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:28.617+0000] {cursor.py:804} INFO - query: [DROP TABLE IF EXISTS GOLD_ANALYSIS.MARCHE.TEMP_FAIT_PRIX_OR]
[2025-05-12T17:17:28.750+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:28.751+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:28.753+0000] {logging_mixin.py:151} INFO - Table Fait_Prix_Or mise à jour avec succès dans Snowflake (sans doublons)
[2025-05-12T17:17:28.755+0000] {cursor.py:804} INFO - query: [ROLLBACK]
[2025-05-12T17:17:28.849+0000] {cursor.py:817} INFO - query execution done
[2025-05-12T17:17:28.851+0000] {cursor.py:959} INFO - Number of results in first chunk: 1
[2025-05-12T17:17:28.853+0000] {logging_mixin.py:151} INFO - Connexion Snowflake fermée
[2025-05-12T17:17:28.854+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-12T17:17:28.873+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=gold_pipeline, task_id=load_gold_data, execution_date=20250512T171619, start_date=20250512T171719, end_date=20250512T171728
[2025-05-12T17:17:28.926+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-12T17:17:28.944+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
